app/assets/javascripts/student_profile_v2/risk_bubble.js

```diff
@@ -0,0 +1,56 @@
+(function() {
+  window.shared || (window.shared = {});
+  var dom = window.shared.ReactHelpers.dom;
+  var createEl = window.shared.ReactHelpers.createEl;
+  var merge = window.shared.ReactHelpers.merge;
+
+  var styles = {
+    riskBubble: {
+      fontSize: 20,
+      width: '35px',
+      height: '35px',
```

Comment:
For React styles, you can use plain numbers here (eg., `35` instead of `35px` like you would in CSS).  React will always interpret numbers as being pixels.